### PR TITLE
manifest: update Zephyr revision

### PR DESCRIPTION
**Pull request by VeijoPesonen**
Brings in LwM2M v1.1 related changes.

* New content formats
* Networking fixes

Depends on https://github.com/nrfconnect/sdk-zephyr/pull/789

Signed-off-by: Veijo Pesonen <veijo.pesonen@nordicsemi.no>

### CODE REVIEW

**Comment by VeijoPesonen** [comment]
@jheiskan81, @rlubos For your information.

**Comment by VeijoPesonen** [comment]
We are still to add LwM2M sample config overlays which enable the features introduced to Zephyr's LwM2M client.

**Comment by jheiskan81** [comment]
I added Overlay updates and Conformance test enablers.

File: samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1-core-interop.conf [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+CONFIG_LWM2M_VERSION_1_1=y
+CONFIG_LWM2M_RW_OMA_TLV_SUPPORT=y
+CONFIG_LWM2M_COAP_MAX_MSG_SIZE=1280
+CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE=10
+
+#Enable SenML-JSON content format
+CONFIG_BASE64=y
+CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
+
+#Enable SebML-CBOR content format
```
Comment (VeijoPesonen) [inline samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1-core-interop.conf:30]:
```suggestion
#Enable SenML-CBOR content format
```

File: samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1-core-interop.conf [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+CONFIG_LWM2M_RW_SENML_CBOR_SUPPORT=y
+CONFIG_LWM2M_RW_SENML_CBOR_RECORDS=45
+CONFIG_ZCBOR=y
+CONFIG_ZCBOR_CANONICAL=y
+
+#Enable extra content format
```
Comment (VeijoPesonen) [inline samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1-core-interop.conf:36]:
```suggestion
#Enable legacy JSON content format
```

File: samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1-core-interop.conf [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+CONFIG_LWM2M_RW_JSON_SUPPORT=y
+
+#Enable dynamic Send Operation
+CONFIG_APP_LWM2M_CONFORMANCE_TESTING=y
+
+#Modem lib trace's control
+CONFIG_NRF_MODEM_LIB_TRACE_ENABLED=n
```
Comment (VeijoPesonen) [inline samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1-core-interop.conf:43]: Not needed at all so should be removed

File: samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1-object-interop.conf [hunk 4]
```diff
@@ -0,0 +1,40 @@
+#Disable Aplication sensor Object and GNSS modules away
```
Comment (VeijoPesonen) [inline samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1-object-interop.conf:1]: Same comments as with [overlay-lwm2m-1.1-core-interop.conf](https://github.com/nrfconnect/sdk-nrf/pull/7546/files#diff-68388808842a48a6c459f8bb72d6e7f83b1f84484dc298b66fe1630dc674a9e5)

File: samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1.conf [hunk 5]
```diff
@@ -1,5 +1,12 @@
 CONFIG_BASE64=y
 CONFIG_LWM2M_VERSION_1_1=y
+CONFIG_LWM2M_RW_OMA_TLV_SUPPORT=y
 CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
 CONFIG_LWM2M_COAP_MAX_MSG_SIZE=1024
 CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE=10
+
+#Enable SebML-CBOR content format
```
Comment (VeijoPesonen) [inline samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1.conf:8]:
```suggestion
#Enable SenML-CBOR content format
```

File: samples/nrf9160/lwm2m_client/src/main.c [hunk 6]
```diff
@@ -209,6 +296,15 @@ static void rd_client_event(struct lwm2m_ctx *client, enum lwm2m_rd_client_event
 	case LWM2M_RD_CLIENT_EVENT_REGISTRATION_COMPLETE:
 		reconnection_counter = 0;
 		LOG_DBG("Registration complete");
+		if (update_session_lifetime) {
+			/* Read a current nserver lifetime value */
```
Comment (VeijoPesonen) [inline samples/nrf9160/lwm2m_client/src/main.c:300]:
```suggestion
			/* Read a current server lifetime value */
```

**Comment by jheiskan81** [comment]
@VeijoPesonen I fix your comment's

File: west.yml [hunk 7]
```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 79b77b4f6130d0db539a6f6655b3e03818ab98a6
+      revision: pull/789/head
```
Comment (rlubos) [inline west.yml:55]: Please update SHA: `4ab692bec88a143f89f2abfdc086e3f8087622f4`

Comment (jheiskan81) [reply west.yml]: Ok Rebased and Updated.

**Review by rlubos — APPROVED** [review]

**Pull request merged**